_data/users.yml

```diff
@@ -122,6 +122,20 @@
   links:
     - urltext: View the Bazaar Technologies website
       url: https://www.bazaar-tech.com/
+- name: Bloomberg
+  highlight: false
```

Comment:
I assume this keeps them at the forefront of the first page? Perhaps Bloomberg should be on this list me thinks.

Reply:
They will show up as sponsor for. We will also see about users selection, but thats a separate effort.